server/src/test/scala/cromwell/engine/workflow/lifecycle/execution/ejea/EjeaMultipleCallCacheCopyAttemptsSpec.scala

```diff
@@ -158,3 +185,16 @@ class EjeaMultipleCallCacheCopyAttemptsSpec
       ejha = Some(helper.ejhaProbe.ref)))
 
 }
+
+object EjeaMultipleCallCacheCopyAttemptsSpec extends StrictLogging {
+  /*
+  During the BT-140 investigation the `EjeaMultipleCallCacheCopyAttemptsSpec` was seen not responding in one of the test
+  runs. This could have been a GC pause, Akka TestKit not timing out as expected, or any number of other reasons. For
+  now, we're adding a little bit more debug information to this spec just in case this happens again in the future. If
+  at any point someone decides that this is not actually helping, feel free to delete all calls to this method and the
+  debug method itself.
+   */
+  private def bt140Debug(message: String): Unit = {
+    logger.info("BT-140 debug: " + message)
+  }
```

Comment:
I really like and appreciate how you clearly linked this back to the ticket for the specific problem we're trying to debug!